### PR TITLE
s-tui: 1.0.1 -> 1.1.3

### PR DESCRIPTION
**Pull request by liff**
###### Description of changes

A few releases have happened:
 - [1.0.2](https://github.com/amanusk/s-tui/releases/tag/v1.0.2)
 - [1.1.1](https://github.com/amanusk/s-tui/releases/tag/v1.1.1)
 - [1.1.3](https://github.com/amanusk/s-tui/releases/tag/v1.1.3)

Also added an update script and a simple test. Removed `LC_ALL` since it seems to work just fine without.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Kranzes** [comment]
Result of `nixpkgs-review pr 165937` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>s-tui</li>
  </ul>
</details>

**Review by Kranzes — APPROVED** [review]

**Comment by bobby285271** [comment]
s-tui.passthru.tests fails on x86_64-linux.

```
/build/.attr-0: line 1: jq: command not found
```

**Comment by liff** [comment]
> `/build/.attr-0: line 1: jq: command not found`

Oops, thanks. Moved `nativeBuildInputs` to the right place.

File: pkgs/tools/system/s-tui/default.nix [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+  passthru = {
+    updateScript = nix-update-script { attrPath = pname; };
+
+    # Crashes in aarch64 sandbox due lack of CPU frequency info.
+    tests = if stdenv.isAarch64 then { } else {
+      json-output = runCommand "${pname}-test" { nativeBuildInputs = [ jq gnugrep ]; } ''
+        ${s-tui}/bin/s-tui --json | jq -r .Util.Avg > "$out"
+        egrep '^[[:digit:]]+(\.[[:digit:]]+)?$' "$out"
+      '';
+    };
```
[the remark this answers is not in the document]
Comment (liff) [reply pkgs/tools/system/s-tui/default.nix]: Replaced with `testVersion`.

File: pkgs/tools/system/s-tui/default.nix [hunk 2]
```diff
@@ -1,25 +1,36 @@
-{ lib, python3Packages }:
+{ lib
+, stdenv
+, python3Packages
+, runCommand
```
Comment (bobby285271) [inline pkgs/tools/system/s-tui/default.nix:4]:
```suggestion
```

Comment (liff) [reply pkgs/tools/system/s-tui/default.nix]: Oops, thanks.

**Pull request merged**